src/HotChocolate/Core/src/Types.Scalars/SignedByteType.cs

```diff
@@ -0,0 +1,72 @@
+using HotChocolate.Language;
+
+namespace HotChocolate.Types
+{
+    /// <summary>
+    /// The UnsignedInt scalar type represents a unsigned numeric non‐fractional
```

Comment:
this comment still seems to be off ... UnsignedInt but we are a SignedByte

Reply:
yeah ... reading things carefully is a valuable trait ...
done